bubble_sort.rb

```diff
@@ -0,0 +1,16 @@
+def bubble_sort(array)
+  temp_value = 0
+  while temp_value != nil
+    temp_value = nil
+    (array.length - 1).times do |idx|
+      if array[idx] > array[idx + 1]
```

Comment:
To swap 2 elements, `a` and `b` in ruby - `a, b = b, a`.This will eliminate the use of `temp` variable.